src/main/resources/javascript/getAllElementsByPath.js

```diff
@@ -87,6 +87,9 @@ function transform(node) {
 		extractedAttributes[attributeName] = attributeValue;
 	}
 	
+	if (node.checked) {
```

Comment:
Why check? You dont do it for the other attributes...

Reply:
'checked' is only an attribute of inputs that are of type checkboxes... and that cannot easily be expressed in our defaults.yaml. So we do it that way.

Reply:
Yes, same as readonly, disabled, multiple, ... and other "boolean"-attributes, that are added just via`<tag {$attribute}>`

Reply:
For disabled, readonly and others, we can have default attributes in the defaults.yaml. Because we know for which tags they can appear. But `checked` attribute can appear on some instances of input and not others. So whatever way you put the defaults, there's always an edge case that doesn't fit. So we solve this in the JavaScript. That was actually Daniels idea...